native/cocos/renderer/gfx-vulkan/VKDevice.cpp

```diff
@@ -504,11 +502,6 @@ void CCVKDevice::doDestroy() {
 
     SPIRVUtils::getInstance()->destroy();
 
-    for (CCVKTexture *texture : _depthStencilTextures) {
-        CC_SAFE_DESTROY_AND_DELETE(texture)
-    }
-    _depthStencilTextures.clear();
-
     if (_gpuDevice) {
```

Comment:
We might still call `_depthStencilTextures.clear()` to keep the current destruction order.
In some cases, destruction order is purposefully designed by the author, so changing order might result in bugs.

Reply:
I deleted the `_depthStencilTextures` in `VKDevice`. It doesn't seem to be used.

Reply:
I see.